### ISSUE TITLE
DQM content and directory

### ISSUE THREAD

**Issue by ThorbenQuast** [HGCDAQ/eudaq#57]
What we ultimately want is to have an **extended** DQM with respect to the default EUDAQ one.

Content of visualisation:
For monitoring the DATURA telescope data, it has been very useful to have the default collections included. For the tests at CERN in 2017, we already had an own collection for the DWC data and will likely use that again in 2018.
There is no reason to have any of them excluded in our DQM. One may always match the collection to the data type (="planes") by their name. 
E.g. in the HexaBoard converter, one may add: 
``` 
if (pl.Sensor().find("HexaBoard")==std::string::npos)
    return;
```
Directory:
As such a DQM should not only display HGCAL hexagon related data but possibly DATURA telescope, DWC reconstructed data, ... 
Hence, it would be better if the HGC DQM code was decoupled from the HGCAL producer as there are no dependencies to the external software that the producer requires. (Only the HexaBoard conversion plugin should remain in the producer's directory.)
E.g., a suitable location would be /eudaq/monitors/cmshgcal_onlinemon.

A solution for these points has already been implemented and has successfully been used during the beam test at DESY in March 2018. 
The corresponding branch is "dev-thorbenquast".
However, due to the "ad'hoc upgrade" to EUDAQ 1.7 in the tb2018 branch, "dev-thorbenquast" (nor any other 2017 branch) cannot be merged.

**Comment by andreypz** [comment]
Ok, 
Can you do the work of merging the codes? I suggest the following:
* Start with **tb2018** branch and create a fresh directory at *eudaq/monitors/cmshgcal_onlinemon* with a copy of your code. 
* Make it compile as another onlinemon executable, like *HgcOnlineMon2.exe* in addition to existing one.
* Diff all the files of your monitor and the one under 'cmshgcal/onlinemon' and port missing changes (I don't think there are going to be many differences).

Then we can switch to the new monitor. 
I assume it will contain the correlation plots as well, which would be useful.

**Comment by andreypz** [comment]
I merged #60 with your changes.
I tested it by running in offline mode using input files from */disk2_2TB/testRun_orm* (run 271 for example).

Next steps I think should be:
* Remove the onlinemon directory from producers/cmshgcal and make *HgcOnlineMon2.exe* to be *HgcOnlineMon.exe*
* Add wire chambers lines to the config file and run the wire chamber fake data producer together with hgcal.

I don't think we should move the converter plugins to *main/lib/* yet. I think it's more intuitive to find them in the corresponding producer directory. Perhaps in the future, when we switch to eudag 2.0, we can move them, so that we can even merge our code with the main eudaq repository.
